src/data/content/design-system/componenti/chips.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
 
 seo:
   name: Chips - Designers Italia
-  description: "Elementi compatti che rappresentano un input, attributo o azione"
+  description: "Il componente Chips permette di etichettare, classificare e organizzare contenuti ed elementi dell'interfaccia utilizzando parole chiave (tag), stati o altri attributi per abilitare percorsi di navigazione trasversale o filtri e scelte dell'utente."
```

Comment:
```suggestion
  description: "Il componente Chips permette di etichettare, classificare e organizzare contenuti o elementi di interfaccia mediante parole chiave (tag), stati o altri attributi per abilitare percorsi di navigazione trasversale, filtri o scelte dell'utente."
```

Reply:
@Fupete ho revisionato in "contenuti o elementi" invece di "contenuti e elementi", valuta tu se è corretto cmque